feat: Add Danish layout

Cheesebaron opened the pull request:
This PR adds a Danish layout inspired by the Swedish one, swapping some of the characters with Danish ones.

I've run the debug sample and it appears to work:
![Screenshot_20240329_185617](https://github.com/Julow/Unexpected-Keyboard/assets/249719/b60026ed-8833-4987-a186-9c480d014e87)

This is my first PR in this repo, let me know if I missed anything.

Fixes #605

Julow commented on srcs/layouts/latn_qwerty_da.xml:
```diff
@@ -0,0 +1,41 @@
+    <key key0="b" key2=";" key4=","/>
+    <key key0="n" key2=":" key4="."/>
+    <key key0="m" key2="-" key4="_"/>
+    <key width="1.0" key0="." key1="!" key2="\?" key3="'" key4=","/>
```
Some keys are present twice on the layout: `!, ', ,, -, ., ?`.

You can check some properties of the layout with `gradle test`. This will update the file `check_layout.output`, that should be committed.

Cheesebaron replied:
I will have a look at this. Thanks!

Cheesebaron replied:
Should be resolved now!

Julow reviewed (commented):
Nice!

As your screen is quite wide, let me add a screenshot from a smaller device:
![shot](https://github.com/Julow/Unexpected-Keyboard/assets/2310568/6955ab6b-3265-4d5b-a495-1fa78899f2cf)

In my testing, it's usable but maybe not on even smaller screens.

Cheesebaron commented:
![Screenshot_20240331_013430_GitHub.jpg](https://github.com/Julow/Unexpected-Keyboard/assets/249719/722e6d6c-448f-4766-bd8d-c670c878a084)

This is how gboard looks as well. Just FYI.

Julow commented:
The 11-columns layout works for me! Did you have a look at the symbols appearing twice?

Cheesebaron commented:
I've adjusted the layout a bit and removed dupes. Looks like this now:

![image](https://github.com/Julow/Unexpected-Keyboard/assets/249719/e8d194f7-f955-4839-a200-0fcd6ec8237d)

Julow reviewed (approved):
Perfect, thanks! I'll merge now but of course feel free to make modifications in the future.
